crossbar/_logging.py

```diff
@@ -274,12 +273,7 @@ def _make_json(_event):
             # This is a traceback. Print it.
             traceback = event["log_failure"].getTraceback()
 
-            if not six.PY3:
-                traceback = traceback.decode('utf-8')
-                linesep = os.linesep.decode('utf-8')
-            else:
-                linesep = os.linesep
-
+            linesep = os.linesep
             eventText = eventText + linesep + traceback
```

Comment:
Maybe we can just replace `linesep` with `os.linesep` throughout this code (i.e. drop the local)?

Reply:
done, it seems there was only one instance, other places in that file are already using os.linesep